xml/System.Net/HttpWebRequest.xml

```diff
@@ -102,6 +102,15 @@
       </AssemblyInfo>
       <Docs>
         <summary>Initializes a new instance of the <see cref="T:System.Net.HttpWebRequest" /> class.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+
+Both <xref:System.Net.HttpWebRequest> constructors are obsolete and should not be used. Call the <xref:System.Net.WebRequest.Create%2A?displayProperty=nameWithType> method to initialize new <xref:System.Net.HttpWebRequest> objects. If the scheme for the Uniform Resource Identifier (URI) that is provided as an argument to the method is `http://` or `https://`, the <xref:System.Net.WebRequest.Create%2A?displayProperty=nameWithType> method returns an <xref:System.Net.HttpWebRequest> object.    
```

Comment:
Using WebRequest.Create() is ok.  But if you're trying to replace using the HttpWebRequest() constructor, it is better to use WebRequest.CreateHttp().